google/cloud/spanner_v1/batch.py

```diff
@@ -146,7 +147,9 @@ def _check_state(self):
         if self.committed is not None:
             raise ValueError("Batch already committed")
 
-    def commit(self, return_commit_stats=False, request_options=None):
+    def commit(
+        self, return_commit_stats=False, request_options=None, max_commit_delay_ms=None
```

Comment:
What is the expected input for `max_commit_delay_ms` here? Is it a number?

Instead, I would prefer getting input as `datatime.timedelta`. What do you think?

Reply:
We do something similar for `max_staleness` field. Reference: https://github.com/googleapis/python-spanner/blob/07a02023d588b780ca760280be311e9d221a4a41/google/cloud/spanner_v1/snapshot.py#L691

Reply:
I feel this way client library changes will be very minimal

Reply:
Good idea - done.